webapp/src/app/course-welcome/course-welcome.component.ts

```
@@ -140,6 +210,293 @@
+      
+
+      return this.API_URL + course.url.replace(/\\/g, '/');
+    }
+    // Return an empty string or a default image if needed.
+    //return '/assets/img/courseCard.png';
+    return '';
+  }
+
+
+  openFileSelector(): void {
+    if (this.fileUploader && this.fileUploader.inputFile) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)